src/writing-policies/index.md

```diff
@@ -1,21 +1,19 @@
 # Writing Policies
 
 Chimera policies can be written using any kind of language capable of building
-[Wasm](https://webassembly.org/) binaries for the [WASI](https://wasi.dev/) target.
+[Wasm](https://webassembly.org/) binaries and that supports waPC guest SDK.
 
 The list of languages that can produce Wasm modules is continuously evolving.
 [This page](https://github.com/appcypher/awesome-wasm-langs) provides a
 nice overview.
 
-> **Note well:** some languages have Wasm support, but are not yet capable
-> of building binaries for the WASI interface.
->
-> Wasm modules built for the browser target won't run inside of a WASI-compliant
-> runtime.
+It's possible to check what languages support waPC guest by searching
+on the [official github
+group](https://github.com/wapc?q=guest&type=&language=).
 
 ## What a policy does
 
-The Chimera admission controller receives
+The Chimera admission server receives
```

Comment:
```suggestion
The Chimera admission controller receives
```

or webhook works too

Reply:
I would replace that with `Chimera policy server`